src/Inciter/AMR/Refinement_State.h

```diff
@@ -0,0 +1,167 @@
+//
+// Created by Robert Francis Bird on 10/27/16.
+//
+
+#ifndef TETAMR_AMR_DATA_H
+#define TETAMR_AMR_DATA_H
+
+#include <vector>
+#include <cstddef>
+
+#include "types.h"
+
+namespace AMR {
+    // Types
+    using real_t = tk::real;
```

Comment:
On second thought, I think it is actually a good idea to have a separate types header for AMR. Let's call it `AMRTypes.h` though, or something similar.